### PR TITLE
feat: finish Homework 1

### PR DESCRIPTION
**Pull request by ptmdmusique**
# Description

- Add solutions for 32 CSS games
- Finish palindrome with bootstrap styling

### CODE REVIEW

File: hw1/02-palindrome.html [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
+  <script defer src="02-palindrome.js"></script>
+</head>
+
+<body class="vh-100">
+  <main class="bg-primary h-100 pt-4">
+    <section class="card mx-auto w-50 p-3">
+      <h1>Exercise 02 - Palindrome</h1>
+
+      <div class="mt-4">
+        <label>Enter a positive number:</label>
```
Comment (shrutidaundkar) [inline hw1/02-palindrome.html]: This label will generate an alert on Wave saying orphaned label.

File: hw1/02-palindrome.html [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+        <input type="number" />
```
Comment (shrutidaundkar) [inline hw1/02-palindrome.html]: This input tag will generate an error on Wave saying "Form control does not have a corresponding label". To fix both the label and input error on Wave add the "id" attribute in the input tag and the "for" attribute in the label tag.

Comment (ptmdmusique) [reply hw1/02-palindrome.html]: ~~I don't think using `id` is a good idea in general for forms like this, I'll use `name` and `for` instead.~~ Thanks for the suggestion!

Comment (shrutidaundkar) [reply hw1/02-palindrome.html]: You are welcome! Just make sure you check for errors and alerts on the Wave before turning your assignment in.

**Pull request merged**